lib/redis.js

```diff
@@ -0,0 +1,25 @@
+export default class Redis {
+  constructor (options) {
+    this._redis = require('redis').createClient(options.url )
+  }
+
+  createHealthCheck () {
+    return (callback, that = this) => {
```

Comment:
il faut eviter ce genre de chose `that = this`
en l'occurence, l'utilisation de `this` devrais suffire, sinon tu peux faire appel a la fonction `bind`